### DISCUSSION TITLE
windows exporter after a while stops to stream data to a netdata agent on linux

### DISCUSSION THREAD

**Discussion by tradexsrl** [netdata/netdata#18009] [Q&A]
Hi , i have installed the latest windows exporter on 2 vm (MS server 2016 and windows 10) , then i get the guid as explained in the documentation , then i another linux server i modify the file in go.d/windows.conf as in the documentation and then i modify the file in vnodes/vnodes.conf
i restarted the netdata agent on that linux machine and i get the data but after a while a get this situation:
![immagine](https://github.com/netdata/netdata/assets/62424179/de1e9203-9c7a-487f-ab80-83b84de670e9)

![immagine](https://github.com/netdata/netdata/assets/62424179/bc7bb077-57ab-4b9c-9966-300301600639)

and if i try to add a chart to my custom dashboard , when i select these nodes i cannot select disk or others , only data related to anomaly....
please let me know.

**Comment by tradexsrl** [comment]
any news?
i need to implement urgently netdata agent on a large number of windows server but i need to be sure it works 100%

**Comment by tradexsrl** [comment]
anyone can help?

**Reply by ilyam8** [reply]
Hi, @tradexsrl. Empty charts and no node info **in the Nodes tab** are expected when using Netdata + windows_exporter. You will see metrics if you click on the node (Node view).

We are working on [native Agent for Windows](https://github.com/netdata/netdata/releases/tag/v1.46.0#v1460-release-highlights-windows-support), it is a work in progress and we hope to release it this summer. But it is not ready yet.

**Comment by tradexsrl** [comment]
Understand....what cause confusion for example in windows disk stats is the word anomaly in the top left corner:
![immagine](https://github.com/netdata/netdata/assets/62424179/d4f22950-8129-4d2a-9ce5-689f082da01c)
but if you look deeper you can understand that this chart is for free/used disk space
if the native windows agent will be better than the actual prometheus exporter , i'll wait until you release this summer.
